packages/contracts/src/Game.sol

```diff
@@ -537,6 +538,29 @@ contract Game {
         }
     }
 
+    // Helper function to pack cards into field elements.
+    function bytePacking(uint256[] memory deck) internal pure returns (uint256[2] memory packedCards) {
+        // pad the cards to 62 cards (255 for null value)
+        // 31 cards is packed into one single field element
+        uint256[] memory cards = new uint256[](62);
+        // TODO: we assume deck length is less than 62 (since MAX_DECK_SIZE is 40)
+        // but would be nice to have an additional check
```

Comment:
Yeah, normally that will be checked upstream, we could add a `require(deck.length <= 62)` here.